websites/E/Everskies/dist/metadata.json

```
@@ -0,0 +1,35 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Everskies",
+    "author": {
+      "name": "SlowLife",
+      "id": "374905512661221377"
+    },
+    "category": "socials",
+    "logo": "https://i.imgur.com/MnsPDQu.png",
+    "thumbnail": "https://i.imgur.com/scedVmN.png",
+    "version": "1.0.0",
+    "color": "#c9bfff",
+    "tags": [
+      "everskies",
+      "forums"
+    ],
+    "url": "everskies.com",
+    "description": {
+      "en": "Everskies is a dress-up community with focus on creativity!"
```

```suggestion
      "en": "A community site for teens and young adults with focus on creativity!"
```
I literally think that's their official desc like on twitter 🤷

---

idk